src/app/pages/card-import/card-import.page.ts

```diff
@@ -133,7 +133,7 @@ export class CardImportPage implements OnInit {
     console.trace('Saving card');
     this.card = {
       frontImg: this.frontImg,
-      backImg: this.backImg,
+      backImg: this.backImg || null,
```

Comment:
This shouldn't be strictly necessary; if `this.backImg` is `null` it'll add a `null` here.

Reply:
Exactly. But it wasn't saving the card unless I took a picture of the back. The code seems to be expecting there will be a backImg captured, but when there isn't it gets...befuddled?